src/pages/Room/RoomPage/hooks/useMarker.ts

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from 'react';
+import { LandListResponse } from 'api/room/entity';
+import MarkerIcon from 'components/Room/MarkerIcon/index';
```

Comment:
```suggestion
import MarkerIcon from 'components/Room/MarkerIcon';
```
index는 넣을 필요가 없어보입니다.

Reply:
수정했습니다. [e65efff](https://github.com/BCSDLab/KOIN_WEB_RECODE/pull/34/commits/e65efff446063f0b01ede034afed1869afb45508)